templates/taxi/car_confirm_delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <h1>You want delete {{ car }}</h1>
```

Comment:
There is a grammatical error in the heading. It should be 'Do you want to delete {{ car }}?' to make it grammatically correct.